src/test/java/CheckerTest.java

```diff
@@ -0,0 +1,65 @@
+import duke.Checker;
+import duke.DukeException;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CheckerTest {
+
+    @Test
+    public void checkerTest() throws DukeException {
```

Comment:
I like how you extracted all the exception checking into one class instead of the checking them 1 by 1 in the parser class. This also makes this bit of code clean! 👍